tests/e2e/utils/payments.js

```diff
@@ -32,16 +32,16 @@ export async function fillCardDetails( page, card ) {
 		await cardDateInput.type( card.expires.month + card.expires.year, {
 			delay: 20,
 		} );
+		await page.waitFor( 1000 );
 
 		const cardCvcInput = await stripeFrame.waitForSelector(
 			'[name="cvc"]'
 		);
 		await cardCvcInput.type( card.cvc, { delay: 20 } );
+		await page.waitFor( 1000 );
 
-		if ( null !== ( await page.$( '#add_payment_method' ) ) ) {
```

Comment:
This is not a good check because this just checks whether the form is displayed.
The displayed zip code field depends on the country selected (US), and we are currently not changing the country field in our tests. In my case, "France" was selected by default and the zip field was not present, hence the test failed.